export/securedrop_export/print/print_dialog.py

```diff
@@ -0,0 +1,54 @@
+import logging
+
+import gi
+
+gi.require_version("Gtk", "4.0")
```

Comment:
Does this need to be in Python at runtime? Can we express it in the Debian package dependency?